src/main/kotlin/recce/server/RecceServer.kt

```diff
@@ -24,7 +24,7 @@ private val logger = KotlinLogging.logger {}
     externalDocs = ExternalDocumentation(description = "Server Docs", url = "$GITHUB_PROJECT/README.md"),
     security = [SecurityRequirement(name = "basicAuth")]
 )
-@SecuritySchemes(SecurityScheme(name = "basicAuth", type = SecuritySchemeType.HTTP, scheme = "basic"))
+@SecuritySchemes(SecurityScheme(paramName = "Authorization", name = "basicAuth", type = SecuritySchemeType.HTTP, scheme = "basic"))
```

Comment:
In theory, according to swagger docs, this is only supposed to apply to `apiKey` type so it's a bit weird, but at least this fixes it.

Looks like they are changing stuff for `4.6.0` based on https://github.com/micronaut-projects/micronaut-openapi/issues/811 and https://github.com/micronaut-projects/micronaut-openapi/pull/819 (we are on `4.5.2`) but this is probably fine to move forward with.